collect_app/src/main/java/org/odk/collect/android/workers/SmsSenderWorker.java

```diff
@@ -0,0 +1,28 @@
+package org.odk.collect.android.workers;
+
+import android.support.annotation.NonNull;
+
+import org.odk.collect.android.tasks.sms.SmsSender;
+
+import androidx.work.Worker;
+
+/***
+ * Background job that adheres to the fire and forget architecture pattern
+ * where it's sole purpose is to send an SMS message to a destination without
+ * caring about it's response.
```

Comment:
In both cases, "it's" should be "its" because they can't be replaced by "it is" and are instead possessives.

Are you sure it's fire and forget? I would expect that to mean no value at all were returned.

Reply:
True, I was thinking of fire and forget in the context of this `Worker` not getting a response, not the entire app. The app does get a response from the `BroadcastReceiver` so if the whole fire and forget notion isn't fitting it can be removed 😅

Reply:
`In both cases, "it's" should be "its" because they can't be replaced by "it is" and are instead possessives.` 

Yes, this comment was from the `SmsSenderJob` from way before I registered mentally the diff between the two. Thanks. will review the other comments as well.

Reply:
I'm not sure! It seems the worker does get a notion of success or failure back. I would expect fire and forget to mean there's something like an event fired that may trigger some behavior but whatever fired the event never hears back at all. What reference are you using for pattern names?

Reply:
True, yes the `Worker` gets an event that the message has been sent so, to be honest, it's not required there. Some of the patterns I knew of from using message patterns when doing a bit of enterprise backend dev. I am realizing the ideas might be similar but not necessarily applicable in a mobile architecture.